server/models/message.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require( 'mongoose' );
+
+const messageSchema = new mongoose.Schema( {
+    text: {
+        type: String,
+        required: true
+    },
```

Comment:
Let's also add a `read` status to each message

Reply:
add a read field to message model